src/drive/styles/filelist.styl

```diff
@@ -355,12 +357,19 @@ column-width-thumbnail-bigger = 7rem
         flex       0 0 3rem
         padding    0
 
-    .fil-content-cell.fil-content-file
-        padding-right 0
+    .fil-content-cell
+        &.fil-content-status
+            padding 0
 
-    .fil-content-cell.fil-content-file-select
-        display none
-        padding .9375rem 0 .9375rem .9375rem
+        &.fil-content-file-action
+            padding 0
+
+        &.fil-content-file
+            padding-right 0
```

Comment:
Since you rewrite this part, can't you use `u-pr-0` instead of creating a new css class?

Reply:
En fait je suis pas trop fan de rajouter `u-pr-0` et donc une nouvelle logique (les class utilitaires) là où `fil-content-file` est utilisé, je préfère laisser comme ça. Quand on attaquera le chantier de refacto de la liste de fichier, je pense que ce sera plus simple de ne traiter qu'avec une seule façon de faire partout